src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
> If am not wrong , you are suggesting workItem.relationalData.creator.attributes.userName to be replaced by "No creator found" ?

Nope. I think "No creator found" is a terrible UX feedback here.

Also, I don't support filling up `fullName` with username, when `fullName` doesn't exist. Not only because that's counter-intuitive, but it's a borderline false information management.